src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -628,17 +628,14 @@ def _preprocess_response(
 
             # Remove responses without any content
             # These are only observed to happen at the start or end
```

Comment:
```suggestion
```

Reply:
what is this change suggesting? it matches the line above

Reply:
There's a `-` at the start of the line and the background is red, meaning the suggestion is to remove the line.

Izzy's code invalidates the comment. Comment says these are only observed at start or end, this whole PR is about how they can be observed in middle now. Hence removing the comment since it's inaccurate.

Reply:
cool thanks

Reply:
Sorry was not getting emails about these comments, updated.